share/completions/sysbench.fish

```diff
@@ -0,0 +1,111 @@
+### Auto-complete for sysbench (cross-platform and multi-threaded benchmark tool) ###
+
+### sub commands specification ###
+complete -c sysbench -f -a "run\t'Run the test'"
+complete -c sysbench -n "__fish_contains_opt test=fileio" -a "
```

Comment:
I'm willing to bet that `sysbench --test fileio` is also valid, which this won't recognize. You probably want to write a function that iterates over `commandline -opc` and checks if either `--test=$argv` is in there or there's a token `test`, followed by `$argv`.